src/Microsoft.ComponentDetection.Detectors/go/GoComponentDetector.cs

```diff
@@ -52,10 +53,12 @@ protected override async Task OnFileFound(ProcessRequest processRequest, IDictio
             var wasGoCliScanSuccessful = false;
             try
             {
-                if (IsGoCliManuallyEnabled())
+                if (!IsGoCliManuallyDisabled())
                 {
-                    Logger.LogInfo("Go cli scan was manually enabled");
                     wasGoCliScanSuccessful = await UseGoCliToScan(file.Location, singleFileComponentRecorder);
+                } else
+                {
+                    Logger.LogInfo("Go cli scan was manually disabled.");
```

Comment:
"Go cli scan was manually disabled, fallback strategy performed. For more info see https://aka.ms/go-detection-fallback-strategy"

Reply:
updated.